pandas/tests/io/parser/test_textreader.py

```diff
@@ -341,6 +341,16 @@ def test_empty_csv_input(self):
         df = read_csv(StringIO(), chunksize=20, header=None, names=["a", "b", "c"])
         assert isinstance(df, TextFileReader)
 
+    def test_blank_lines_between_header_and_data_rows(self):
+        ref = DataFrame(
```

Comment:
Can you add the GH issue number as a comment?

Reply:
done